src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/checkout/begin.js

```diff
@@ -9,7 +9,7 @@ const { updateSavedCards } = require('*/cartridge/scripts/updateSavedCards');
 
 function shouldRestoreBasket(cachedOrderNumber) {
   // restore cart if order number was cached
-  if (cachedOrderNumber !== undefined) {
+  if (cachedOrderNumber !== undefined && cachedOrderNumber !== null) {
```

Comment:
can't we just check if`cachedOrderNumber` is truthy?

Reply:
Is order number always a String? If so then I'll change it to `if(cachedOrderNumber)`